translation/debug.rst

```diff
@@ -19,9 +19,10 @@ command helps you to find these missing or unused translation messages templates
 
 .. caution::
 
-    The extractors can't find messages translated outside templates, like form
-    labels or controllers. Dynamic translations using variables or expressions
-    in templates are not detected either:
+    The extractors can't find messages translated outside templates (like form
+    labels or controllers) unless using :ref:`translatable-objects` or calling
```

Comment:
Will this find a reference in a different file?

Reply:
It'll be good for the docs team to confirm, but this reference does work if I build the docs locally with Docker, so I think it should work.

Reply:
Yes, this works :)